Backport fix for CVE-2026-108360 (SSRF via redirect in Drawing::setPath) from 5.10.0

diff --git a/src/PhpSpreadsheet/Worksheet/Drawing.php b/src/PhpSpreadsheet/Worksheet/Drawing.php
index 1c33389..bfc38b8 100644
--- a/src/PhpSpreadsheet/Worksheet/Drawing.php
+++ b/src/PhpSpreadsheet/Worksheet/Drawing.php
@@ -139,6 +139,7 @@ class Drawing extends BaseDrawing
             if (str_starts_with($path, 'https:') || str_starts_with($path, 'http:')) {
                 $ctxArray = [
                     'http' => [
+                        'follow_location' => 0,
                         'user_agent' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36',
                         'header' => [
                             //'Connection: keep-alive', // unacceptable performance
